aip/0133.md

```diff
@@ -76,6 +76,28 @@ message CreateBookRequest {
   **should not** contain other optional fields except those described in this
   or another AIP.
 
+### Long-running create
+
+Some resources take longer to create a resource than is reasonable for a
+regular API request. In this situation, the API **should** use a long-running
+operation instead:
+
+```proto
+rpc CreateBook(CreateBookRequest) returns (google.longrunning.Operation) {
+  option (google.api.http) = {
+    post: "/v1/{parent=shelves/*}/books"
+  };
+  option (google.longrunning.operation_info) = {
+    response_type: "Book"
+    metadata_type: "OperationMetadata"
+  }
+}
+```
+
+- The response type **must** be set to resource (what the return type would be
+  if the RPC was not long-running).
+- The `response_type` and `metadata_type` fields both **must** be specified.
```

Comment:
both must => must both? (I'm sure it's correct currently, but sounds odd to me.) Or maybe "Both the `response_rype` and `metadata_type` fields must be specified."

Reply:
(And fix in the same place everywhere else :)

Reply:
Done, thanks. I chose the latter option.